sdk/documentintelligence/test-resources.json

```diff
@@ -89,216 +72,80 @@
-            "defaultValue": {
-                "canonicalizedResource": "[concat('/blob/', parameters('blobStorageAccount'), '/', parameters('labelTablesFixedRowsContainer'), '-v3')]",
-                "signedExpiry": "[dateTimeAdd(utcNow('u'), 'PT3H')]",
-                "signedPermission": "rl",
-                "signedResource": "c"
-            }
-        },
-        "trainingDataClassifier": {
+        "trainingDataClassifierContainer": {
             "type": "string",
             "defaultValue": "training-data-classifier"
         },
         "trainingDataClassifierSasProperties": {
             "type": "object",
             "defaultValue": {
-                "canonicalizedResource": "[concat('/blob/', parameters('blobStorageAccount'), '/', parameters('trainingDataClassifier'))]",
+                "canonicalizedResource": "[concat('/blob/', parameters('blobStorageAccount'), '/', parameters('trainingDataClassifierContainer'))]",
                 "signedExpiry": "[dateTimeAdd(utcNow('u'), 'PT3H')]",
                 "signedPermission": "rl",
                 "signedResource": "c"
             }
         }
     },
     "variables": {
-        "authorizationApiVersion": "2018-09-01-preview",
-        "documentIntelligenceBaseName": "[concat('documentintelligence', parameters('baseName'))]",
-        "documentIntelligenceApiVersion": "2023-10-31-preview",
-        "azureDocumentIntelligenceUrl": "[concat('https://', variables('documentIntelligenceBaseName'), parameters('documentIntelligenceEndpointSuffix'))]",
+        "apiVersion": "2023-10-01-preview",
+        "endpoint": "[format('https://{0}{1}', parameters('baseName'), parameters('cognitiveServicesEndpointSuffix'))]",
         "cognitiveServiceUserRoleId": "[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Authorization/roleDefinitions/a97b65f3-24c7-4388-baec-2e87135dc908')]"
     },
     "resources": [
-        {
-            "type": "Microsoft.Authorization/roleAssignments",
-            "apiVersion": "[variables('authorizationApiVersion')]",
-            "name": "[guid(concat(variables('cognitiveServiceUserRoleId'), variables('documentIntelligenceBaseName')))]",
-            "dependsOn": [
-                "[variables('documentIntelligenceBaseName')]"
-            ],
-            "properties": {
-                "principalId": "[parameters('testApplicationOid')]",
-                "roleDefinitionId": "[variables('cognitiveServiceUserRoleId')]"
-            }
-        },
         {
             "type": "Microsoft.CognitiveServices/accounts",
-            "name": "[variables('documentIntelligenceBaseName')]",
-            "apiVersion": "[variables('documentIntelligenceApiVersion')]",
+            "name": "[parameters('baseName')]",
+            "apiVersion": "[variables('apiVersion')]",
             "sku": {
                 "name": "S0"
             },
-            "kind": "DocumentIntelligence",
+            "kind": "FormRecognizer",
```

Comment:
Shouldnt this stay as DocumentIntelligence?

Reply:
I think "FormRecognizer" is the real name in use. Technically FR and DI are deploying same resource, why the "kind" name here should be different?

Reply:
If they really are still the same then that seems fine, but then why not leave it as DocumentIntelligence? Not blocking this pr though

Reply:
The deployment will fail with kind in "DocumentIntelligence".